bcgov/biohubbc
# SIMSBIOHUB-361: Remove all study areas button

## Links to Jira Tickets

[SIMSBIOHUB-361](https://apps.nrs.gov.bc.ca/int/jira/browse/SIMSBIOHUB-361)

## Description of Changes

- Added 'Remove All Study Area' button to Survey create/edit page
- Updated error message for study areas
- Updated verbiage for site selection formik error
- Fixed an issue on the map when selecting features on the map (strange black border)

## Testing Notes

- Create or modify a survey to add multiple features to the study area
- Click the remove all button 
- See that all drawn or imported features are removed from the list and map
